CHANGELOG.md

```diff
@@ -8,6 +8,10 @@
 - Update the docs for generated interface types to reference the MDN Web Docs
   project.
 - Address several broken links in API documentation.
+- Generate APIs based on if they're on track to be a standard and not
```

Comment:
Not for this PR, but it would be nice to be able to have course descriptions of how the generated API changed between major versions. So, something like:

- added classes `Foo`, `Bar`, and `Baz`
- remove class `Qux`

Perhaps this could be done by iterating over the code_builder model and generating a very high-level overview to a text file? Just the list of classes perhaps. If that was committed to the repo, then a simple diff would give us enough info to write a changelog.